src/constants/index.js

```diff
@@ -5,4 +5,89 @@ export const refShapes = [
   PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
 ];
 
-export default { refShapes };
+export const theme = {
```

Comment:
WDYT about moving theme to a file on its own?

Reply:
Done 👍